aries_cloudagent/protocols/out_of_band/definition.py

```diff
@@ -3,8 +3,8 @@
 versions = [
     {
         "major_version": 1,
-        "minimum_minor_version": 0,
-        "current_minor_version": 0,
+        "minimum_minor_version": 1,
```

Comment:
We want to support minor version 0 for OOB.  Unless it is hugely complex, we **always** want to support all minor versions from 0 to `current_minor_version` to ensure backwards compatibility.